project/Dependencies.scala

```diff
@@ -133,7 +133,7 @@ object Dependencies {
   val pact4sCirce =       "io.github.jbwheatley"  %% "pact4s-circe"     % pact4sV
   val okHttp =            "com.squareup.okhttp3"  % "okhttp"            % "4.12.0"
 
-  val workSpaceManagerV = "0.254.1093-SNAPSHOT"
+  val workSpaceManagerV = "0.254.1121-rt-aami-SNAPSHOT"
```

Comment:
TODO: update after https://github.com/DataBiosphere/terra-workspace-manager/pull/1754 is merged